.eslintrc.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     'testing-library/await-async-query': 'error',
     'testing-library/no-await-sync-query': 'error',
     'testing-library/no-dom-import': 'off',
+    'linebreak-style': 0,
```

Comment:
Curious why this was added @kelvinsekx.

Reply:
Going to remove this, since it seems unrelated and should not block this from being merged.
```suggestion
```